davin_readme.md

```diff
@@ -0,0 +1,23 @@
+Looking at [ConfigRepository.java](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java#L527) it looks like the pointer has moved some.  I assume the method we want here is `writeSourceConnectionNoSecret` now located at [ConfigRepository.java:536](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java#L536)
```

Comment:
close! that's the source function. what we want is the connection which is actually [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-server/src/main/java/io/airbyte/server/handlers/ConnectionsHandler.java#L156) and [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-server/src/main/java/io/airbyte/server/handlers/ConnectionsHandler.java#L224).